### DISCUSSION TITLE
Standup (Tuesday, May 17, 2022)

### DISCUSSION THREAD

**Discussion by CarlosCRG19** [CarlosCRG19/ijalti-peers#65] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - No development, however, i started reviewing firebase as an alternative to handle authentication. 

**What are your priorities for the next 24 hours?**:
 - Talk to @BrandonMagana and define issues
 - Develop first signup and login endpoints. 

**Blockers**:
 - Moving out to another apartment, been a little bit busy

**Shoutouts**:
 - @everyone. You're so cool guys.

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
 - Just revisit the overall project

**What are your priorities for the next 24 hours?**:
 - Planning with frontend team

**Blockers**:
 - We need to schedule a meeting (front)

**Shoutouts**:
 - @everyone for getting back in the action

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Looked for ways to style the form

**What are your priorities for the next 24 hours?**:
 - Define styling and start chip components

**Blockers**:
 - I don't understand MUI styling lol

**Shoutouts**:
 - @antonieto for being such a good stepbro

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Read a little about authentication with firebase and heroku

**What are your priorities for the next 24 hours?**:
 - Experiment with Authentication

**Blockers**:
 - None

**Shoutouts**:
 - @antonieto  for moving to another apartment

**Comment by paotg123** [comment]
**What did you achieve in the last 24 hours?**:
 - Looked for ways to style the form

**What are your priorities for the next 24 hours?**:
 - have a meeting with front

**Blockers**:
 -

**Shoutouts**:
 - @antonieto @CarlosCRG19
